### ISSUE TITLE
Monitoring KL loss from VariationalDense

### ISSUE THREAD

**Issue by sbaurdlp** [EderSantana/seya#55]
Hi Eder,

I see that your VariationalDense layer implements the KL loss as a regularizer, how can I monitor it during training ?

Thank you for your help,

Sebastien

**Comment by EderSantana** [comment]
That is a bit tricky… I would have to check out how Keras monitoring has been working recently.

Lately, the way I write and monitor variational layers now is the way I did on commaai/research https://github.com/commaai/research/blob/master/models/autoencoder.py#L203-L205 

in other words, I use keras as helpers where possible, but I also use tensorflow directly

**Comment by sbaurdlp** [comment]
And so the function ```tf.scalar_summary``` allow to monitor scalar values in TensorBoard ?

**Comment by sbaurdlp** [comment]
Thank you, it seems to be exactly what I need, I'll have a look
